.github/workflows/merge-changes.yaml

```
@@ -0,0 +1,46 @@
+name: Cherry-pick and Create PR
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+    inputs:
+      base_branch:
+        description: "Name of the base branch who last commit should be cherry picked and applied to the starting branch"
+        required: true
+        type: string
+      starting_branch:
+        description: "Name of the starting branch from which the cherry picked branch of the base should be applied to"
```

I'm not 100% sure about this one so please double-check this suggestion:
```suggestion
        description: "Name of the starting branch where the cherry picked commit of the base should be applied to"
```